web/src/app/components/Scoreboard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PlayerState from '../interfaces/PlayerState';
+import ProgressBar from './ProgressBar';
+import playerColors from '@/scripts/PlayerColors';
+
+interface ScoreboardProps {
+    players: PlayerState
+    numWords: number;
+}
+
+export default function ScoreboardProps({players, numWords}: ScoreboardProps) {
+  return (
+    <>
```

Comment:
Why do you always do this? Idk if it breaks anything but shouldn't we be just using divs?

Reply:
I don't think it really matters, it's more of a style thing. I see a lot of react examples using this syntax